lib/ui/platform_dispatcher.dart

```diff
@@ -968,21 +971,29 @@ class PlatformDispatcher {
   /// This option is used by [showTimePicker].
   bool get alwaysUse24HourFormat => _configuration.alwaysUse24HourFormat;
 
-  /// The system-reported text scale.
-  ///
-  /// This establishes the text scaling factor to use when rendering text,
-  /// according to the user's platform preferences.
-  ///
-  /// The [onTextScaleFactorChanged] callback is called whenever this value
-  /// changes.
-  ///
-  /// See also:
+  /// Deprecated. Will be removed in a future version of Flutter.
   ///
-  ///  * [WidgetsBindingObserver], for a mechanism at the widgets layer to
-  ///    observe when this value changes.
+  /// Use [platformTextScaler] instead.
+  @Deprecated(
+    'Use platformTextScaler instead. '
+    'This feature was deprecated after 3.11.0-5.0.pre.',
+  )
   double get textScaleFactor => _configuration.textScaleFactor;
 
-  /// A callback that is invoked whenever [textScaleFactor] changes value.
+  /// Deprecated. Will be removed in a future version of Flutter.
+  ///
+  /// This field is renamed to [onTextScalerChanged].
```

Comment:
nit: it is not just a rename, right? The semantics also changed, presumably, it fires when textScaler changes and not when textscalefactor changes?